services/src/main/java/org/keycloak/broker/saml/mappers/AdvancedAttributeToGroupMapper.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.broker.saml.mappers;
+
+/**
+ * @author <a href="mailto:denis.bernard@avanade.com">Denis Bernard</a>
```

Comment:
Not here?

Reply:
Sorry, just like other classes you are adding, the `@author` should be right above the class declaration.